app/poros/google_user.rb

```diff
@@ -0,0 +1,15 @@
+class GoogleUser
+  attr_reader :google_id,
```

Comment:
We'll need to add the new attributes for household, incomplete chores and completed chores